zeppelin-web/src/app/notebook/paragraph/paragraph.css

```diff
@@ -396,6 +396,11 @@ table.table-shortcut {
  opacity: 0 !important;
 }
 
+#main .emacs-mode .ace_cursor {
+  background: #C0C0C0!important;
```

Comment:
+1 for overriding emacs mode only while keeping the same appearance with normal mode.